Code/GraphMol/ChemTransforms/catch_tests.cpp

```diff
@@ -56,3 +58,202 @@ TEST_CASE("Github #1039", "[]") {
   }
 }
 
+TEST_CASE("molzip", "[]") {
+    SECTION("basic tests")
+    {
+        auto a = "C[*:1]"_smiles;
+        auto b = "N[*:1]"_smiles;
+        auto mol = molzip(*a,*b);
+        CHECK(MolToSmiles(*mol) == "CN");
+    }
+    
+    {
+        auto a = "C[*]"_smiles;
```

Comment:
not sure what the `Isotope` is doing in this example.
Again, would probably help to have some more examples that include some edge cases

Reply:
It is the label to extract from the isotope which in this case is 0.  I could enforce it to have to be set.

Reply:
yeah, the semantics in the rest of the RDKit is more or less that `0` means that it hasn't been set.

Reply:
Ok, I'll make the change for this.